terraform/modules/cloud/aws/deploy/fleet/templates/user_data.bash

```diff
@@ -26,15 +26,12 @@ sudo service datadog-agent start
 
 #INSTALL EPOCH
 
-VERSION="${epoch_version}"
+URL="${epoch_package}"
 
-FILE="epoch-$VERSION-ubuntu-x86_64.tar.gz"
-URL="https://github.com/aeternity/epoch/releases/download/v$VERSION/$FILE"
-
-wget -q $URL -O /home/epoch/$FILE
+wget -q $URL -O /home/epoch/epoch.tar.gz
```

Comment:
Above, value of variable `URL` is assigned with quotes `"` while here no quotes used: shall quotes be here too?

I see name is inject from other config file, is URL, and potentially may contain strange characters. Though it does not matter for now so looks ok.